src/DataAccessLayer/Repositories/StudentRepositoryCSV.cs

```diff
@@ -0,0 +1,89 @@
+﻿using CsvHelper;
+using DataAccessLayer.Entities;
+using DataAccessLayer.Interfaces;
+using OfficeOpenXml;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+
+namespace DataAccessLayer.Repositories
+{
+    public class StudentRepositoryCSV : IRepository
+    {
+        public void Create(IEnumerable<StudentToWrite> item, double averageGroup, string path)
```

Comment:
It is not CSV! CSV could not contain named cells ("A1") and so on.
Move to create a method to specific Excel service.